### ISSUE TITLE
Premium plugin version/update not up to date 

### ISSUE THREAD

**Issue by Domisiding** [SpiGetOrg/Spiget#25]
Hey
If I query Premium Plugins via the API, I do not get the latest version, but mostly those that are several weeks old. The same is the case with updates, the newest ones are not output on latest.

[API Message](https://api.spiget.org/v2/resources/49554/versions/latest)

Version of the Plugin: 2.2.5

**Comment by InventivetalentDev** [comment]
Yeah premium resources are a bit of a mess to keep updated, due to spigotmc's cloudflare blocking a lot of the requests for those.

**Comment by Domisiding** [comment]
Hello, it is possible to query this via google's cache.  The problem would not be with these.

**Comment by InventivetalentDev** [comment]
Hmm good point actually, Didn't realize they had a public cache available, will look into that!

**Comment by Domisiding** [comment]
The descriptions could thus also be displayed. By the Cache Methode

**Comment by InventivetalentDev** [comment]
Looks like there's a pretty strict limit to making requests to that cache, so won't be able to use that either.

**Comment by Domisiding** [comment]
If you pretend to be a Google Bot, a direct query would actually be possible or?

**Comment by InventivetalentDev** [comment]
Pretty sure they're smarter than that.

**Comment by InventivetalentDev** [comment]
For the record: It did pretend to be a googlebot for a while which is actually how I got premium resource data working in the first place, but that got blocked by CF a couple months ago, so I had to switch back to regular User-Agents.

**Comment by Domisiding** [comment]
A query from such an address is not often granted? The possibilities are kept pretty small 

https://webcache.googleusercontent.com/search?q=cache:a_TmFp4ZgBUJ:https://www.spigotmc.org/resources/x-replay-advanced-replay-plugin.49554/+&cd=1&hl=de&ct=clnk&gl=de&client=firefox-b-dhttps://webcache.googleusercontent.com/search?q=cache:a_TmFp4ZgBUJ:https://www.spigotmc.org/resources/x-replay-advanced-replay-plugin.49554/+&cd=1&hl=de&ct=clnk&gl=de&client=firefox-b-d

**Comment by Domisiding** [comment]
Could you contact Spigot and leave the server's whitelisted by CF? That the query goes through without being blocked

**Comment by Domisiding** [comment]
Könnte man nicht die User_Agents von Facebook, Twitter etc benutzen, da diese frei sind und wenn Spigot einen von diesen Blockt auf ein anderen Switchen.
